package.json

```diff
@@ -66,6 +66,6 @@
     "axios-mock-adapter": "^1.20.0",
     "glob": "7.2.3",
     "husky": "7.0.4",
-    "jest": "29.5.0"
+    "jest": "^26.6.3"
```

Comment:
Why is it necessary to downgrade this version? @AuraAlba

Reply:
I got an error in test for dates and was for mismatch of jest version, that is the one @edx/frontend-build uses and where we get the configuration for jest